packages/canary/src/styles.css

```diff
@@ -11,31 +11,34 @@
 
 :root {
   /* PP Figma shades */
-  --black: 210 10% 3%;
-  --grey-6: 210 8% 6%;
-  --grey-8: 210 7% 8%;
-  --grey-10: 210 6% 10%;
-  --grey-12: 210 6% 12%;
-  --grey-15: 210 6% 15%;
-  --grey-20: 210 6% 20%;
-  --grey-30: 210 6% 30%;
-  --grey-40: 210 6% 40%;
-  --grey-50: 210 6% 50%;
-  --grey-60: 210 6% 60%;
-  --grey-70: 210 6% 70%;
-  --grey-80: 210 6% 80%;
-  --grey-90: 210 6% 90%;
-  --grey-94: 210 6% 94%;
-  --grey-98: 210 6% 98%;
+  --black: 240 10% 3%;
+  --grey-6: 240 8% 6%;
+  --grey-8: 240 7% 8%;
+  --grey-10: 240 6% 10%;
+  --grey-12: 240 6% 12%;
+  --grey-15: 240 6% 15%;
+  --grey-20: 240 6% 20%;
+  --grey-30: 240 6% 30%;
+  --grey-40: 240 6% 40%;
+  --grey-50: 240 6% 50%;
+  --grey-60: 240 6% 60%;
+  --grey-70: 240 6% 70%;
+  --grey-80: 240 6% 80%;
+  --grey-90: 240 6% 90%;
+  --grey-94: 240 6% 94%;
+  --grey-98: 240 6% 98%;
   --white: 0 0% 100%;
 
   /* PP Figma secondary colors */
   --mint: 175 60% 65%;
   --blue: 216 80% 65%;
   --orange: 35, 75%, 55%;
   --purple: 275 60% 75%;
-  --green: 150, 75%, 65%, 0.7;
-  --red: 0, 80%, 65%, 0.7;
+  --green: 150, 75%, 65%;
+  --red: 0, 80%, 65%;
+  --red-glow: 0, 80%, 65% 0.5;
+  --orange-glow: 35, 75%, 55% 0.5;
+  --green-glow: 150, 75%, 65% 0.5;
```

Comment:
we decided to not hardcode opacity into base colors. we already have the same `--green` color.. opacity/alpha should be applied when using that color via tailwind